FirebaseUI/Database/API/FirebaseArrayDelegate.h

```diff
@@ -36,7 +38,7 @@
  * @param object The object added to the FirebaseArray
  * @param index The index the child was added at
  */
-- (void)childAdded:(id)object atIndex:(NSUInteger)index;
+- (void)array:(FirebaseArray *)array didAddObject:(id)object atIndex:(NSUInteger)index;
```

Comment:
This change is pretty impactful I think, people do use FirebaseArray directly. We may want a migration cheat sheet - replace X with Y kind of thing.